driver/bpf/missing_definitions.h

```diff
@@ -0,0 +1,39 @@
+/*
+
+Copyright (C) 2023 The Falco Authors.
+
+This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+or GPL2.txt for full copies of the license.
+
+*/
+
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 16, 0)
+struct ovl_entry {
```

Comment:
I imagine this is not accessible in any kernel header, is it true? Since we are defining our own copy of them I would suggest adding a prefix like `bpf__` to these structs, in this way, we won't never clash against some already defined struct and it would be easier for us to immediately understand this is a struct defined by us

Reply:
Same for the flag enum, and in the kmod

Reply:
Just noticed you have already tested the patch on some kernels, since this is a nitpick let's leave it as it is now